data_processors/pipeline/lambdas/bcl_convert.py

```diff
@@ -331,7 +332,9 @@ def handler(event, context) -> dict:
     # All good, add as input
     workflow_input['settings_by_samples'] = settings_by_samples
 
-    workflow_engine_parameters = wfl_helper.get_engine_parameters(target_id=seq_name)
+    portal_run_uuid = get_tiny_uuid()
```

Comment:
Could we change like `liborca.get_tiny_uuid()` instead? Just minor but it helps when refactor.

Reply:
> Do you have bandwidth for the remaining TODOs?
I could use a hand with the migration and tests (2 and 4). Happy to do 1 & 3

Reply:
Sure, sound good. I will probably continue work on this branch. So, when you are happy with 1 & 3, could you please ping me. Then, I will take over the branch. 

Also, you may leave those _teeny-weeny_ comments that I made; to me. I can follow up on those. 😉

Reply:
1 & 3 complete -

Reply:
Might just see if I can update the output file prefix and output directory values for each workflow too to ensure they're consistent

Reply:
Looking at this, actually secondary_target_id is not needed since the top output is the workflow run uid, can have a quick chat if you want to make sense of this

Reply:
> Looking at this, actually secondary_target_id is not needed since the top output is the workflow run uid, can have a quick chat if you want to make sense of this

Contradicting myself within a minute here - it is needed for tso ctdna step since top directory is 'results' and for consistency across secondary analysis pipelines this should be `library-id/results`, but for all other workflows, they have an output structure that contains the library name in the top output folder.

Reply:
Okay! Let me work on the branch and, let you know if any... 😅

Reply:
I guess we can reset this conversation with https://github.com/umccr/data-portal-apis/pull/352/commits/26a395058575d43ff5dc7441b603b1b14789b486